### DISCUSSION TITLE
github.com/PHP-CS-Fixer/doc/usage.rst#environment-options: not found

### DISCUSSION THREAD

**Discussion by 3xploiton3** [PHP-CS-Fixer/PHP-CS-Fixer#7374] [Q&A] [answered]
![image](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/19517680/0de6c2b4-277d-4995-864f-ca1b20692a92)

**Comment by Wirone** [comment, marked as answer]
I believe it's related to #5860, right?
